Directory.Build.props

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <VersionPrefix>4.12.1</VersionPrefix>
+    <!-- TODO: Remove Suffix for 5.0.0 release   -->
+    <VersionSuffix>alpha</VersionSuffix>
```

Comment:
@bruno-garcia you [commented here](https://github.com/getsentry/sentry-dotnet/pull/3513#discussion_r1798648332):
> Should we just add this by hand when releasing>?

You mean when running the Release workflow in GitHub?

Reply:
yeah, but it's fine to leave it here too if you prefer